### DISCUSSION TITLE
What's the difference and which one to choose?

### DISCUSSION THREAD

**Discussion by rafnov** [organicmaps/organicmaps#9821] [Q&A] [answered]
I recently dropped onto 2 versions of Organic Maps on Droid-ify which connects to F-droid and IzzyOnDroid.
I've seen one on Google Store too.
What's the difference between them while version number is the same? What's the matter of the size difference? 
Why the size of Google Store version looks twice as big as others?
Which one should I choose and use and why?

![Screenshot_2024-12-04-22-09-41-36_bf5286d428c97f865e4b51a5afbaefcd.jpg](https://github.com/user-attachments/assets/ec3f0cdc-7222-43b8-a873-658c7093f3bf)

![Screenshot_2024-12-04-22-09-49-06_bf5286d428c97f865e4b51a5afbaefcd.jpg](https://github.com/user-attachments/assets/7c4abf7e-6558-477f-a4d3-cfd4e5285e5a)

![Screenshot_2024-12-04-22-28-16-92_da477480df70e1f8b6688bd2fd977143.jpg](https://github.com/user-attachments/assets/36692b92-15a0-4fba-9ddf-972afd1c731d)

**Comment by Misalf-git** [comment, marked as answer]
`version` | Google | Web | F-Droid
:----: | :----: | :----: | :----:
Can use Google location services | ✅ | ✅ | ❌
Includes World map | ✅ | ❌ | ❌

**Reply by rafnov** [reply]
Got another question: if F-droid version can't use Google location services, then what it can use?

**Reply by map-per** [reply]
It can directly use the phones GPS (without taking e.g. WiFi or cellar network into consideration as Google location services do it).

**Reply by rafnov** [reply]
I still don't get the thing.
Why it says Google in the name, but installed and recommended version looks from F-droid?
Both circled red on the screenshot.

![IMG_20241213_153232.jpg](https://github.com/user-attachments/assets/b35c8349-3a1c-439c-b8f8-d125d8213f4a)

**Reply by hb0nd** [reply]
>  Why it says Google in the name, but installed and recommended version looks from F-droid?

The app versions all have the same ID (app.organicmaps) so your google install is appearing in FDroid (that's the version id)

**Reply by rafnov** [reply]
Ok, thx to you all, guys, I got some additional info, but I finally managed to check them all myself. My summary is:
1. *-Google contains all - basic world map and Google location
2. *-Web contains Google location, but still needs to download basic world map
3. *-F-droid doesn't rely on Google location, but on GPS sensor only (?).

Still # 3 works pretty accurately (I disabled Wi-Fi and mobile data to test), which was proved by a short walk :), but do I really need # 1? Maybe I am not aware of some things related to Google location services?

The # 2 is pointless as it requires to download basic world map anyway and still uses Google services.
So the choice is between # 1 and # 3.
Sorry if I bother you to much, just need to understand.
Btw. All versions use the same amount of data to download and save on a device, so really only Google location is under my consideration for yes or no.

**Reply by Misalf-git** [reply]
I'm using the "Web" version from GitHub because...
- app (and map data) updates are available earlier via GitHub.
- on my device, the GPS allone is usually quick enough, but it might happen that I need to rely on network/cellular location depending on where I am or current weather.
